include/MidiPort.h

```diff
@@ -174,6 +174,8 @@ private slots:
 	Map m_readablePorts;
 	Map m_writablePorts;
 
+	QMap<int16_t, int16_t> m_activeMIDIPitch;
```

Comment:
In the interest of not using Qt in the core, ``m_activeMidiPitch`` can become a ``std::unordered_map``.
```suggestion
	std::unordered_map<int16_t, int16_t> m_activeMIDIPitch;
```